portfolio/src/main/webapp/styles/styles.css

```diff
@@ -0,0 +1,118 @@
+body {
+ font-family: 'Work Sans', sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+ font-family: 'Crimson Text', serif;
+}
+
+main {
+    display: flex;
+    flex-basis: 50%;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    height: auto;
+    margin: 1em auto;
+}
+
+.content {
+    display: flex;
+    border: solid 0.5em transparent;
+    flex-direction: column;
+    width: 40%;
+    height: auto;
+    text-align: left;
+}
+
+.title {
+    color: #0c8716;
+    font-size: 4em;
+    margin: 0;
+}
+
+#pronounce {
+  width: 1.5em;
```

Comment:
nit: indentation in this file should be consistent

Reply:
will do